docs/connecting.txt

```diff
@@ -89,29 +89,17 @@ connection with the database superuser ``crate``.
     properties.put("ssl", "true");
     Connection conn = DriverManager.getConnection("crate://localhost:5432/", properties);
 
-In addition connection properties can be passed via the JDBC URL:
+In addition, connection parameters can be passed via the JDBC URL:
 
 .. code-block:: java
 
     Connection conn = DriverManager.getConnection("crate://localhost:5432/?property1=value1&property2=value2");
 
-CrateDB JDBC driver supports following properties:
+.. NOTE::
 
-:strict:
-    **Default**: ``false``.
+   The CrateDB JDBC driver supports all of `the PostgreSQL JDBC driver
+   parameters`_, though not all of them will make sense for CrateDB.
```

Comment:
I think we should come up with a list of parameters that we think are relevant and have effect to the connection with CrateDB.

Reply:
Relevant properties provided from the pgjdbc upstream are:
* user
* password
* ssl
* loglevel
* prepareThreshold
* loginTimeout
* connectTimeout
* socketTimeout
* tcpKeepAlive
* readOnly
* currentSchema
* loadBalanceHosts

Reply:
@chaudum should we just list these and link through to the parent documentation, or should we duplicate the parent documentation? or something crate specific?

I am concerned that the documentation for `strict` is so verbose. it is going to make a list of either non-documented or minimally documented parameters look out-of-place

Reply:
I would duplicate the description of relevant parameters and additionally add the link to pgjdbc (full reference).

`strict` is a kind of an important parameter, it's ok if the description is verbose.